### PR TITLE
Atexit in init

### PR DESCRIPTION
**Pull request by Dominik-Vogel**
adding Instrument.close_all() and UMR to init.
works with Ipython and through spyder.

### CODE REVIEW

**Comment by jenshnielsen** [comment]
This will need to wait for https://github.com/QCoDeS/Qcodes/pull/1009 otherwise :+1:

**Comment by Dominik-Vogel** [comment]
Hang on, I think this should better go into core... what do you think @jenshnielsen ?

**Comment by jenshnielsen** [comment]
Yes I think so

**Comment by Dominik-Vogel** [comment]
moved atexit part to 
https://github.com/QCoDeS/Qcodes/pull/1017
left UMR

File: qdev_wrappers/__init__.py [hunk 1]
```diff
@@ -2,4 +2,9 @@
 from qdev_wrappers.device_annotator.device_image import save_device_image
 from qdev_wrappers.show_num import show_num
 from qdev_wrappers.sweep_functions import do0d, do1d, do2d, do1dDiagonal
+
 from qcodes.monitor.monitor import Monitor
+from qcodes.instrument.base import Instrument
```
Comment (jenshnielsen) [inline qdev_wrappers/__init__.py]: I think this is unused now?

**Comment by jenshnielsen** [comment]
Should we wrap this in a try, except Import error to support old version of qcodes? Otherwise :+1:

File: qdev_wrappers/__init__.py (hunk 1 above)
Comment (Dominik-Vogel) [reply qdev_wrappers/__init__.py]: This might break things if people are not importing the instrument correctly...

Comment (jenshnielsen) [reply qdev_wrappers/__init__.py]: Ok fine to leave it

**Pull request merged**